br-webui/views/network.liquid

```diff
@@ -181,21 +181,85 @@ function isValidIP(ip) {
 }
 
 function setCompanionEthernetIP() {
-	var ip_addr = prompt("Please enter a new IP Address");
+	var value = document.getElementById("current-config").innerText;
+	console.log(value);
+	var mode = value.trim();
+	console.log(mode);
+	if (mode === "Manual") {
+		var ip_addr = prompt("Please enter a new IP Address in the 192.168.2.x subnet");
+
+		var valid_ip = isValidIP(ip_addr)
+		if (!valid_ip) {
+			alert("Invalid IP address: " + ip_addr);
+			return
+		}
+	
+		if (confirm("Are you sure you want to change the ethernet IP to " + ip_addr + "?")) {
+			socket.emit('set default ip', ip_addr);
+		}
+		alert("New IP will take affect after reboot");
+	}
+	else {
+		alert("Changing IP address is only allowed in manual mode");
+	}
+
+}
```

Comment:
semicolon

Reply:
There isnt supposed to be a semicolon over here.

Reply:
Per the BR department guideline: semicolons are required after every statement

Statement definition: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements

However after further reading on the subject, it looks like it's not best practice to place after code blocks in {}. I will update the guideline.
https://stackoverflow.com/questions/17036135/why-are-semicolons-not-used-after-if-else-statements
https://stackoverflow.com/questions/36091659/using-semi-colons-to-close-functions-in-javascript-necessary

please copy and paste the function definition here:
http://www.jslint.com/

You will find some other problems that are at odds with our guidelines.